pandas/tests/io/parser/common/test_common_basic.py

```diff
@@ -862,3 +862,27 @@ def test_read_seek(all_parsers):
             actual = parser.read_csv(file)
         expected = parser.read_csv(StringIO(content))
     tm.assert_frame_equal(actual, expected)
+
+
+@pytest.mark.xfail
+def test_read_csv_skip_blank_rows(all_parsers):
+    # GH22693
+    parser = all_parsers
+    csv_f = StringIO(
+        """A, B, C, D
+    FOO, 1, 2, 3
+    FOO, 4, 5, 6
+    , , ,
```

Comment:
Sorry should have been clearer. I meant moving it completely to the left, """ allows this. Please also remove the whitespaces in the blank line completely